zerver/tests/test_widgets.py

```diff
@@ -152,7 +152,7 @@ def test_todo(self) -> None:
 
         sender = self.example_user('cordelia')
         stream_name = 'Verona'
-        content = '/todo'
+        content = '/todo \nget sunlight - green grass\nx trim lawn - red lawnmower  \n\n   get milk\n - get eggs'
```

Comment:
Do we want to borrow the common Markdown-style syntax of `- [x]` va. `- [ ]` for TODO lists?  That seems more likely to match what we would do in a native markdown feature.